IsThereAnyDeal.psm1

```diff
@@ -5,12 +5,19 @@ function ConvertGamesToITAD ($allGames) {
         $status = $games.CompletionStatus | Sort-Object | Select-Object -Last 1
         @{
             title = $games[0].Name
-            status = if ($status) {([string]$status).ToLower()} else {$null}
-            playtime = $playtime / 60
+            # status = if ($status) {([string]$status).ToLower()} else {$null}
```

Comment:
Is there a reason why you want to get rid of tracking status? I know playtime barely shows up in the ITAD UI, I'd be fine getting rid of it, but status actually works and shows in the collection list well.

The one issue I can think of is that if you want to keep it up to date, you have to manually reupload your collection periodically because this extension doesn't currently sync periodically.

Reply:
Mainly because I'd rather track status on Playnite, actually, and unless it's via auto-sync I'd rather not have the same concept on 2 different platforms. Also, in my particular case the assortment of "status" does not match ITAD combo box at all.

I see your point, though. Would you consider making it optional to track status and play time?